config/routes.rb

```diff
@@ -78,6 +78,7 @@
       post 'mturk_auto/update_cron', to: 'mturk_auto#update_cron', as: 'mturk_auto_update_cron'
       post 'mturk_auto/update_primary_jobs', to: 'mturk_auto#update_primary_jobs', as: 'mturk_auto_update_primary_jobs'
       get 'mturk_auto/evaluate_batch/:mturk_auto_batch_id', to: 'mturk_auto#evaluate_batch', as: 'mturk_auto_evaluate_batch'
+      resources :primary_mturk_batch_jobs
```

Comment:
The controller does not define `show` or `edit`. Should we exclude those from valid routes?

Reply:
Yep, will do.